CDP4CrossViewEditor/Views/UserControls/ThingSelector.xaml

```diff
@@ -0,0 +1,31 @@
+﻿<uc:ThingUserControl
+    x:Class="CDP4CrossViewEditor.Views.UserControls.ThingSelector"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:uc="clr-namespace:CDP4CrossViewEditor.Views.UserControls"
+    DataContextChanged="ThingUserControl_DataContextChanged"
+    mc:Ignorable="d">
+    <GroupBox x:Name="GroupBoxThing" Padding="5" VerticalAlignment="Stretch" HorizontalAlignment="Stretch">
+        <Grid>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="35*" />
+                <ColumnDefinition Width="15*" />
+                <ColumnDefinition Width="50*" />
+            </Grid.ColumnDefinitions>
+            <StackPanel Grid.Row="0" Grid.Column="0">
+                <uc:ThingSelectorSource />
```

Comment:
For the future, I don't think that it was strictly speaking necessary to break this down further into 3 distinct controls as they do not typically function without each other. The source, selector and the target could have been all in this xaml.

Reply:
Understood.